apps/nextjs-website/src/app/auth/confirmation/page.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+import { Box, CircularProgress, Stack } from '@mui/material';
+import { environment } from '@/config';
+import PageNotFound from '@/app/not-found';
+import { Auth } from 'aws-amplify';
+import { useSearchParams, useRouter } from 'next/navigation';
+import { useEffect, useState } from 'react';
+
+const Confirmation = () => {
+  const searchParams = useSearchParams();
+  const router = useRouter();
+  const username = searchParams.get('username');
+  const code = searchParams.get('code');
+
+  const [renderNotFound, setRenderNotFound] = useState(environment === 'prod');
```

Comment:
```suggestion
  const [renderNotFound, setRenderNotFound] = useState(isProduction);
```
I think here you could use the `isProduction` comring from the `@/config`